Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2743,6 +2743,22 @@ public Expression applyTargetMap(Expression source, String targetMap) {
             String functionArgument = targetMap.substring(invocationStart + 1, targetMap.lastIndexOf(')'));
             Expression argumentSource =
                     functionArgument.equals("%value") ? source : applyTargetMap(source, functionArgument);
+
+            // NOTE: This is needed to work around the mapping for ToInterval
```

Comment:
General feedback:  Please fix the title and description of this PR so at a minimum it's not a truncated version of the commit message.